src/Popover/Popover.tsx

```diff
@@ -145,13 +215,13 @@ export default class Popover extends React.Component {
   // UTILS
 
   // extend with default values
-  getPopoverProps = (_props) => {
+  getPopoverProps = (_props?: PopoverProps) => {
     const props = _props || this.props;
     return {
-      type: 'relative',
-      position: 'top',
-      anchor: 'center',
-      event: 'hover',
+      // type: 'relative', //FIXME(gabro): doesn't seem used
```

Comment:
double check, it wasn't defined in the props in the `.d.ts` files and I didn't find any usage.

Reply:
this can be removed: the prop `type` has been replaced by `attachToBody`